src/fsipy/automatedPostprocessing/log_plotter.py

```diff
@@ -505,6 +508,11 @@ def plot_probe_points(time: np.ndarray, probe_points: Dict[int, Dict[str, np.nda
         ax2.legend([l1, l2], ["Velocity Magnitude", "Pressure"], loc="upper right")
         ax2.tick_params(axis='y', which='major', labelcolor='r')
 
+    # Remove any empty subplots if the number of probe points doesn't fill the entire grid
+    for i in range(num_selected_probe_points, num_rows * num_cols):
```

Comment:
I tried this new functions but got a following error here.
```
--- Creating plot for velocity magnitude and pressure for probe points 
7
Traceback (most recent call last):
 line 514, in plot_probe_points
    fig.delaxes(axs[i])
IndexError: index 7 is out of bounds for axis 0 with size 4
```

Reply:
If seems like `axs` needs to be flattened? It seems to be very similar to `plot_probe_points_tke` but there’s slight difference how `axs` and `axes` are used.

Reply:
Thanks, this should be fixed now.